corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -67,7 +67,7 @@
   }
 
   .stripe-repeats {
-    .gr {
+    > .row, .panel-body > .children > .row {
       &:nth-of-type(odd) {
```

Comment:
should this target the `.gr` element so the resulting target is the same as before? So like 
```
 &:nth-of-type(odd) {
    .gr {
         background-color: @table-bg-accent;
      }
  }
  ```

Reply:
@nospame fya

Reply:
is there a resolution to this? My guess is we want to target the same elements as before. I think the way it is done now extends past the border that we want it to. I'm not sure if what i suggested does it correctly, can you verify?

Reply:
With current change, it looks like:
![Screenshot from 2024-01-18 14-57-32](https://github.com/dimagi/commcare-hq/assets/14318173/5e7972af-d7bf-48b6-9898-a324e156695d)

With suggested change, it looks like:
![Screenshot from 2024-01-18 14-58-58](https://github.com/dimagi/commcare-hq/assets/14318173/78c92a81-102b-40bd-a498-ef120cde1795)

In this, there seem to be a border cut-off from left and right.

Reply:
I see, hm i think currently it would also be cut-off on the left and right. I think the full width does look better. @nospame do you have an opinion? I see in the screenshot of the original [PR](https://github.com/dimagi/commcare-hq/pull/33844) that there's also a left and right gap. Wanted to check it wasn't for a particular reason.

Reply:
@Jtang-1 my bad on not keeping up on Github notifications. No, there's no particular reason for the left and right gap, that was just how it worked out with the elements I was able to easily/logically target.